public/Aryan Suri/Citation/citation.css

```diff
@@ -0,0 +1,44 @@
+#citeModal {
+    display: flex;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: none;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+    justify-content: center;
+    align-content: center;
+}
+
+#citeModalContent {
+    background-color: #fefefe;
+    padding: 20px;
+    border: 1px solid #888;
+    width: 40vw;
+    margin-left: 520px;
+    z-index: 30000001;
+    top: calc(50% - 150px);
+    position: absolute;
+    border-radius: 10px;
+}
+
+#citeContent {
+    margin-top: 10px;
+    display: flex;
+    justify-content: space-between;
+}
+
+#citeText {
+    font-size: 1rem;
+}
+
+#citeSelect {
+    max-width: 30%;
+}
+
+#citeCopy:hover, #citeCopyHTML:hover, #citeBIBTEX:hover {
+    font-style: italic !important;
+}
```

Comment:
Great job with proactively choosing specific id names! This will greatly limit the possibility of future CSS conflicts.